website/static/js/home-page/newAndNoteworthyPlugin.js

```diff
@@ -117,7 +117,9 @@ var NewAndNoteworthy = {
         }
 
         function findMoreProjectsButton () {
-            return m('a.btn.btn-default.m-v-lg', {type:'button', href:'/search'}, 'Search for more projects');
+            return m('a.btn.btn-default.m-v-lg', {type:'button', href:'/search', onclick: function() {
+                $osf.trackClick('newAndNoteworthy', 'navigate', 'navigate-to-search-for-more-projects');
```

Comment:
As discussed over flowdock, please change `newAndNoteworty` to `discoverPublicProjects` so that it's clear where these logged events are happening.